### PR TITLE
Decidims Finder Rebased

### PR DESCRIPTION
**Pull request by dperezserra**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -13,5 +13,5 @@
 //= require jquery
 //= require rails-ujs
 //= require activestorage
-//= require_tree .
+//= require cable
```
Comment (tramuntanal) [inline app/assets/javascripts/application.js:16]: can you undo this change?

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -11,6 +11,6 @@
  * It is generally better to create a new file per style scope.
  *
  *= require decidim
- *= require_tree .
+ *= require overrides
```
Comment (tramuntanal) [inline app/assets/stylesheets/application.css:14]: can you undo this change?

**Review by tramuntanal — CHANGES_REQUESTED** [review]
HI @dperezserra,

- can you add all images into an `asset/images/decidims_finder` directory please?
- you probably need translations to `:oc`
- I've also left two comments to avoid introducing unwanted changes

Thanks!

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (spuigmitjadxc) [reply app/assets/javascripts/application.js]: changed to "//= require_tree ./channels" to load successfully the scripts:
```
<script src="https://cdn.jsdelivr.net/npm/algoliasearch@4.5.1/dist/algoliasearch-lite.umd.js"
    integrity="sha256-EXPXz4W6pQgfYY3yTpnDa3OH8/EPn16ciVsPQ/ypsjk=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@4.8.3/dist/instantsearch.production.min.js"
    integrity="sha256-LAGhRRdtVoD6RLo2qDQsU2mp+XVSciKRC8XPOBWmofM=" crossorigin="anonymous"></script>
```
with "//= require_tree ." scripts are not load successfully, for that reason @dperezserra  changed to cable, is there a better way to do it?

File: app/controllers/decidims_finder_controller.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+class DecidimsFinderController < Decidim::ApplicationController
+  layout "layouts/decidim/application"
+
+  helper_method :current_translations
+
+  def show; end
+
+  def current_translations
+    @translations ||= I18n.backend.send(:translations)
+    @translations[I18n.locale].with_indifferent_access
```
Comment (tramuntanal) [inline app/controllers/decidims_finder_controller.rb:12]:
```suggestion
    @current_translations ||= begin
        translations= I18n.backend.send(:translations)
        translations[I18n.locale].with_indifferent_access!
    end
```

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (tramuntanal) [reply app/assets/javascripts/application.js]: It seems you both are right https://stackoverflow.com/questions/39294859/require-tree-argument-must-be-a-directory-in-a-rails-5-upgraded-app/39312176 change accepted!

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Can you change the way to memoize `current_translations` please?

**Review by tramuntanal — APPROVED** [review]
:+1: thanks for the refactor @spuigmitjadxc
